tw/style.css

```diff
@@ -23,43 +25,43 @@ body {
 }
 
 .main-view {
-    margin: 0 15px;
+    margin: 0 !important;
 }
 
 h2 {
-    font-size: 2em;
-    margin-left: 0.5em;
-    margin-top: 1em;
+    font-size: 24px;
+    margin: 0;
     font-style: italic;
 }
 
 .script-category-section {
     background-color: #f5f6fa;
-    /* margin: 0 15px; */
 }
 
 .script-panels-container {
-    margin: -5px;
```

Comment:
Keeping this would prevent panels from touching each other, making them look more has "cards" .